### DISCUSSION TITLE
[BUG] meta fields nav

### DISCUSSION THREAD

**Discussion by nestle49** [timber/timber#2990] [Q&A]
### Expected Behavior

I register meta fields with Carbon Fields

```
<?php

use Carbon_Fields\Container;
use Carbon_Fields\Field;

add_action( 'carbon_fields_register_fields', 'crb_attach_post_meta' );
function crb_attach_post_meta(): void
{
    Container::make( 'nav_menu_item', 'Menu Settings' )
        ->add_fields( array(
            Field::make('image', 'nav_item_icon, 'Icon')->set_width( 100 ),
        ));
}
```

How to use meta fields with Timber::get_menu('primary')? I trying item.meta('nav_item_icon'), but doen't work

### Actual behavior

.

### What version of Timber are you using?

2.1

### What version of WordPress are you using?

WordPress 6.4

### What version of PHP are you using?

8.1

### How did you install Timber?

Composer

**Comment by Levdbas** [comment]
Hi @nestle49 , We do not have an integration with Carbon fields. There was a PR somewhere that made a good start with the integration but that was never finished. Next time, please take the time to fill in the issue template.
